app/styles/ui/_preferences.scss

```diff
@@ -1,4 +1,12 @@
 #preferences {
+  .preferences-container {
+    display: flex;
+    .tab-container {
+      border-left: 1px solid var(--box-border-color);
```

Comment:
there's a variable for this already at https://github.com/desktop/desktop/blob/cdbfb04ff1afda377e59e8e5f158da9f446442d0/app/styles/_variables.scss#L198 that you can use